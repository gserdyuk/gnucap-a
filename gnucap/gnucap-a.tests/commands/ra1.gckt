*

.param a1='1+abs(1)'

v1 1 0 dc 2.1
r1 1 0 10k

.echo message
.printparam 'a1'
.printparam 238.e-3
.printparam 'a1+2.22'

.print op v(1)
.op

.end


