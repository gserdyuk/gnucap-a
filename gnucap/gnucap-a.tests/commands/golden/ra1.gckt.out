Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* 
.echo message 
.printparam {a1}=  2. 
.printparam 238.e-3=  0.238 
.printparam {a1+2.22}=  4.22 
#           v(1)      
 27.        2.1       
